webpack.config.js

```diff
@@ -12,10 +13,21 @@ module.exports = {
     },
     module: {
         loaders: [
-            { test: /\.css$/,  loader: 'style!css' },
+            {
+                test:   /\.css$/,
+                loader: "style!css!postcss-loader"
```

Comment:
Always use [dangling commas](http://eslint.org/docs/rules/comma-dangle) in objects, to minimise code churn and maximise future diffs readability.